css-pseudo-classes/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Pseudo Classes</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h2>Hover Button</h2>
+
+    <form>
+      <label for="button">
+        <button id="button">I'm a button!</button>
+      </label>
+    </form>
```

Comment:
No `form` is required here! We aren't submitting any data, so there's no need for it. You can just use the `button` element on it's own